plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeConfig.java

```diff
@@ -475,4 +485,34 @@ public DeltaLakeConfig setRegisterTableProcedureEnabled(boolean registerTablePro
         this.registerTableProcedureEnabled = registerTableProcedureEnabled;
         return this;
     }
+
+    @Min(value = MIN_READER_VERSION, message = "Must be in between " + MIN_READER_VERSION + " and " + MAX_READER_VERSION)
+    @Max(value = MAX_READER_VERSION, message = "Must be in between " + MIN_READER_VERSION + " and " + MAX_READER_VERSION)
+    public int getReaderVersion()
+    {
+        return readerVersion;
+    }
+
+    @Config("delta.reader-version")
```

Comment:
`delta.default-reader-version` might be better as `delta.default-checkpoint-writing-interval`. What do you think about it?

Reply:
Make sense. I have renamed the property.